azure-cs-cosmosapp-component/Azure.CosmosAppComponent.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Pulumi.Azure" Version="1.4.0-preview" />
+    <PackageReference Include="Pulumi.Docker" Version="1.0.0" />
```

Comment:
This is a fake version that I tested locally with. It needs to change before we merge.

Reply:
@mikhailshilkov can you also update to the correct version of the Pulumi.Docker plugin as well :)